### ISSUE TITLE
Many things in Navigation screen are not operational through a keyboard (Tab) interface.

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#814]
**Describe the bug**
Many things in Navigation screen are not operational through keyboard standard keys. If we want to access different buttons such as " Delete Site", "Copy Site" etc we are not able to access them through Tab key.

**To Reproduce**
1. Go to Navigation screen and select a page for a site to see its navigation.
2. See that all the options/controls on the screen are not accessible using standard Keyboard keys like Tab key.

**Expected behavior**
Whole page should be accessible using Standard keyboard Keys.
